### PR TITLE
Multiple minor fixes and updates

### PR DESCRIPTION
**Pull request by cPanelChrisD**
FIXED - Only report processes that LFD has actually killed (Kill:1)
UPDATED - Change all instances of "case ###" and "FB ###" when mentioning old bug case numbers to "FB-###".
UPDATED - Update help wording for net.ipv4.tcp_tw_{recycle,reuse} to clarify.
UPDATED - Version detection for several fixed Horde SQLite conversion problems.
ADDED - Version detection for MySQL upgrade touchfile check (FB-113349).
ADDED - Version detection for locales check (FB-62119 and FB-107865).
ADDED - Additional EasyApache 4 symlink/permission checks.
ADDED - Check permissions of /var/log directory
UPDATED - /etc/localtime check and help text clarification that it can break All The Things.
ADDED - Ncom rootkit check

### CODE REVIEW

File: ssp [hunk 1]
```diff
@@ -2850,15 +2850,36 @@ sub check_for_non_default_permissions {
...
+        $check{'/usr/local/apache/domlogs'}         = { symlink => '/etc/apache2/logs/domlogs', check_missing => 1 };
+        $check{'/usr/local/apache/error_log'}       = { symlink => '/etc/apache2/logs/error_log', check_missing => 1 };
+        $check{'/usr/local/apache/htdocs'}          = { symlink => '/var/www/html', check_missing => 1 };
+        $check{'/usr/local/apache/logs'}            = { symlink => '/etc/apache2/logs', check_missing => 1 };
+        $check{'/usr/local/apache/modules'}         = { symlink => '/etc/apache2/modules', check_missing => 1 };
+        $check{'/usr/sbin/httpd'}                   = { mode => ['0755'] };
+        $check{'/var/log/apache2'}                  = { mode => ['0711'] };
+        $check{'/var/www/html'}                     = { mode => ['0755'] };
+    } else {
+        $check{'/usr/local/apache/bin/httpd'}       = { mode => ['0755'] };
```
Comment (davelcpanelnet) [inline ssp]: This is actually a symlink

root@centos6 [~]# stat /usr/local/apache/bin/httpd
  File: `/usr/local/apache/bin/httpd' -> `/usr/sbin/httpd'

File: ssp [hunk 2]
```diff
@@ -2850,15 +2850,36 @@ sub check_for_non_default_permissions {
...
+        $check{'/usr/local/apache/domlogs'}         = { mode => ['0711'], perms_help => 'If users can\'t access logs, stats won\'t process.  See ticket 5413079.' };
```
Comment (davelcpanelnet) [inline ssp]: Actually a symlink

root@centos6 [~]# stat /usr/local/apache/domlogs
  File: `/usr/local/apache/domlogs' -> `/etc/apache2/logs/domlogs'

File: ssp [hunk 3]
```diff
@@ -2850,15 +2850,36 @@ sub check_for_non_default_permissions {
...
+        $check{'/usr/local/apache/htdocs'}          = { mode => ['0755'] };
```
Comment (davelcpanelnet) [inline ssp]: Symlink again:

root@centos6 [~]# stat /usr/local/apache/htdocs
  File: `/usr/local/apache/htdocs' -> `/var/www/html'

**Comment by davelcpanelnet** [comment]
root@centos6 [~]# curl https://raw.githubusercontent.com/cPanelChrisD/SSP/master/ssp | perl
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  266k  100  266k    0     0   379k      0 --:--:-- --:--:-- --:--:--  505k

	!!!!! EA4 DETECTED. Deployment is NOT SUPPORTED. !!!!!

	Please DO NOT paste output from SSP into tickets unless it is relevant to an issue
	Please DO NOT paste output from SSP into tickets unless it is relevant to an issue
	Please DO NOT paste output from SSP into tickets unless it is relevant to an issue

	Did you know? The cpanel-postgresql* packages are for phpPgAdmin. The postgresql-* packages are for PostgreSQL

	SSP 4.99.114

[INFO] * Hostname: centos6.davel.local
[INFO] * OS: CentOS release 6.7 (Final) [virtualbox]
[INFO] * Kernel/CPU: 2.6.32-431.el6.x86_64 x86_64 virtualbox Intel Xeon CPU W3520 @ 2.67GHz w/ 1 core(s)
[INFO] * cPanel Info: 11.51.9999.142 (11.51.9999 tier) Last update: 2.3 days ago [ Installed Thu Apr 16 14:54:46 2015 ]
[INFO] * cPanel update check: server is configured to use an unknown tier (11.51.9999)
[INFO] * Uptime:  16:06:17 up 2 days, 11:01,  3 users,  load average: 0.08, 0.02, 0.01
[INFO] * Apache: [ 2.4.16 ] [ Sep  9 2015 14:13:46 w/ ea-apache24-2.4.16-4.3.x86_64 ]
[INFO] * Apache: is listening on ports [ 443 80 ]
[INFO] * PHP Default: 5.6.13 ( ea-php56 ) \w suphp
[INFO] * MySQL Version: mysql  Ver 14.14 Distrib 5.5.45, for Linux (x86_64) using readline 5.1
[INFO] * Backups: [New: Disabled] [Legacy: No Config]
[INFO] * Mailserver: dovecot
[INFO] * FTP Server: pure-ftpd ( Passive ports not enabled )
[WARN] * SELinux: is NOT disabled!
[WARN] * NAT: external IP address 208.74.121.106 is not bound to server
[WARN] * /etc/cpsources.conf: exists! This can affect upcp and EA.
[WARN] * /var/cpanel/nocloudlinux: exists! CloudLinux cannot be installed when this file is present.
RPM check (running "rpm -qa ..."). This will timeout after 25 seconds.
RPM query OpenSSL changelog (running "rpm -q ..."). This will timeout after 15 seconds.
Done.

The checks seem to work so I am not requiring a further commit.  We may want to revise and make mroe accurate checks to validate the /usr/local/apache/* paths being that the majority will be as such:

root@centos6 [~]# ls -la /usr/local/apache/
total 16
drwxr-xr-x.  4 root root 4096 Sep 15 12:22 ./
drwxr-xr-x. 20 root root 4096 Sep 15 12:22 ../
lrwxrwxrwx.  1 root root   28 Sep 15 12:22 access_log -> /etc/apache2/logs/access_log
drwxr-xr-x.  2 root root 4096 Sep 15 12:22 bin/
drwxr-xr-x.  2 root root 4096 Sep 15 12:22 conf/
lrwxrwxrwx.  1 root root   25 Sep 15 12:22 domlogs -> /etc/apache2/logs/domlogs/
lrwxrwxrwx.  1 root root   27 Sep 15 12:22 error_log -> /etc/apache2/logs/error_log
lrwxrwxrwx.  1 root root   13 Sep 15 12:22 htdocs -> /var/www/html/
lrwxrwxrwx.  1 root root   17 Sep 15 12:22 logs -> /etc/apache2/logs/
lrwxrwxrwx.  1 root root   20 Sep 15 12:22 modules -> /etc/apache2/modules/

For now all is sane and works as desired.

reviewed: davelcpanelnet Thu-Sep-17-16:08:23-CDT-2015

File: ssp (hunk 1 above)
Comment (cPanelChrisD) [inline ssp]: I moved that over from the main code-path so it only applies to EA3 and not EA4

File: ssp (hunk 2 above)
Comment (cPanelChrisD) [inline ssp]: Ditto for this line, only affects EA3, so not a symlink.

File: ssp (hunk 3 above)
Comment (cPanelChrisD) [inline ssp]: Ditto previous two notes.

**Pull request merged**